### PR TITLE
UX-327 Video Component

### PR DESCRIPTION
**Pull request by logansparlin**
### What Changed
- Adds new `Video` and `Video.Source` components

### How To Test or Verify
- `npm run start:storybook`
- Visit http://localhost:9001/?path=/story/media-video--basic-video

### PR Checklist

- [x] Add the correct `type` label
- [ ] Pull request approval from #uxfe or #design-guild

### CODE REVIEW

File: stories/media/video.stories.js [hunk 1]
```diff
@@ -1,78 +1,27 @@
...
-  'Analytics-Crop.mp4': require('@sparkpost/matchbox-media/videos/Analytics-Crop.mp4'),
-  'Recipient-Validation.mp4': require('@sparkpost/matchbox-media/videos/Recipient-Validation.mp4'),
-  'Recipient-Validation-Crop.mp4': require('@sparkpost/matchbox-media/videos/Recipient-Validation-Crop.mp4'),
-  'Signup.mp4': require('@sparkpost/matchbox-media/videos/Signup.mp4'),
-  'Signup-Crop.mp4': require('@sparkpost/matchbox-media/videos/Signup-Crop.mp4'),
-};
+export const BasicVideo = withInfo({ propTables: [Video] })(() => (
+  <Video>
+    <Video.Source src={Analytics} type="video/webm" />
+  </Video>
```
Comment (jonambas) [inline stories/media/video.stories.js:14]: Can we add `mp4` sources to these stories so they work on Safari?

**Review by jonambas — CHANGES_REQUESTED** [review]
Just a small story comment, looks good though

File: stories/media/video.stories.js [hunk 2]
```diff
@@ -3,15 +3,29 @@ import { withInfo } from '@storybook/addon-info';
...
 
 export const BasicVideo = withInfo({ propTables: [Video] })(() => (
-  <Video>
-    <Video.Source src={Analytics} type="video/webm" />
-  </Video>
+  <Box display="flex">
+    <Box>
+      <Box as="h4">webm</Box>
+      <Video>
+        <Video.Source src={Analytics} type="video/webm" />
```
Comment (jonambas) [inline stories/media/video.stories.js:19]: You can add two sources here under the same `<Video>` 😬

Comment (logansparlin) [reply stories/media/video.stories.js]: Oh duh 🤦

**Review by jonambas — APPROVED** [review]

**Pull request merged**